docs/audio-peripherals/passthrough/wiring.md

```diff
@@ -35,14 +35,15 @@ Previously, we configured I2S2 for the microphone so we will have to connect the
 {% hint style="info" %}
 TASK 5: From the "Pinout" configuration on CubeMX, determine which pins of the STM32 board are used by I2S2. Using the card provided with the board \(see PDF figure above\), use the connector cables to wire the pins from the STM32 board to the appropriate pins on the microphone breakout board.
 
-_Hint: for example, from the "Pinout" tab we can see that **I2S2\_SD** is outputted on pin **PC3**. From the card provided with the board, we see **PC3** is located in the bottom left corner of the board's pin header extensions. Therefore, we will use a wire to connect this pin to the **DOUT** pin of the microphone breakout board._
+_Hint: for example, from the "Pinout" tab we can see that **I2S2\_SD** is output on pin **PC3**. From the card provided with the board, we see **PC3** is located in the bottom left corner of the board's pin header extensions. Therefore, we will use a wire to connect this pin to the **DOUT** pin of the microphone breakout board._
 {% endhint %}
 
 Finally, we configured an additional GPIO pin in order to select whether we would like the microphone to be assigned to the left or right channel.
 
 {% hint style="info" %}
 TASK 6: Connect the microphone's **SEL** pin to the pin on the STM32 board corresponding to **LR\_SEL**.
 
+<!-- is there a solution for this somewhere? -->
```

Comment:
wouldn't it make sense to add the solution to this task?

Reply:
added the solution for the bonus questions.